pkg/volume/csi/csi_client.go

```diff
@@ -75,7 +75,7 @@ type csiClient interface {
 		ctx context.Context,
 		volID string,
 		targetPath string,
-	) (*volume.Metrics, error)
+	) (*volume.Metrics, *csipbv1.NodeGetVolumeStatsResponse, error)
```

Comment:
Need to add a NodeSupportsVolumeCondition.

Reply:
Instead of returning the entire NodeGetVolumeStatsResponse, can you just return VolumeCondition from NodeGetVolumeStatsResponse here?  Also whether to return nil or VolumeCondition should depends on the feature gate.